.drone.star

```diff
@@ -6,8 +6,8 @@ def get_test_commands():
   return [
     "mkdir -p build",
     "cd build",
-    'CFLAGS="$ARCH_FLAGS" CXXFLAGS="$ARCH_FLAGS" meson ..',
-    "ninja -v",
+    'CFLAGS="$ARCH_FLAGS" CXXFLAGS="$ARCH_FLAGS" meson .. || (cat meson-logs/meson-log.txt; false)',
+    "ninja -v || ninja-build -v",
```

Comment:
For this line, I wanted to write like this to set the value as default: `ninja`. When centos7, the value is set as `ninja-build` by the environment variable. But I recognized the value `${FOO}` is not evaluated for the `{ environment: {"FOO": "something"}` in `.drone.star`. So, I could not adapt the way.

```
"${NINJA:-ninja} -v",
```

Reply:
Perhaps we might be able to just replace `ninja -v` with `ninja-build -v` rather than `ninja -v || ninja-build -v`.

Reply:
With `"ninja -v || ninja-build -v"`, if the build fails because of a compilation problem it will try again with `ninja-build -v`.  How about something like `"\"$(which ninja || which ninja-build)\" -v"` instead?

Just using ninja-build would probably be okay, but if it's not an "official" alias (i.e., an alias installed by ninja not just distros) I'd prefer not to just because I'd like it to work if we ever add a configuration that doesn't support `ninja-build`.  That said, it wouldn't be a big deal.

Reply:
> "\"$(command -v ninja || command -v ninja-build)\" -v" instead?

This looks great! I might want to use the single quote to avoid the escape. `'"$(command -v ninja || command -v ninja-build)" -v'`. Let me fix it.

> i.e., an alias installed by ninja not just distros

You are right. It is not official alias.